src/components/views/rooms/RoomSublist.tsx

```diff
@@ -589,6 +590,16 @@ export default class RoomSublist extends React.Component<IProps, IState> {
                             >
                                 { _t("Show previews of messages") }
                             </StyledMenuItemCheckbox>
+                            <br />
+                            <br />
+                            <ContextMenuButton
+                                kind="primary"
+                                onClick={this.onShowLessClick}
+                                isExpanded={true}
+                                label={_t("Show default amount of rooms")}
+                            >
+                                { _t("Show default amount of rooms") }
```

Comment:
That label brings an element of surprise in my opinion. A user might not be really aware of what the default amount of rooms is. At the moment it seems that this number is `8` and set in `ListLayout#L85`.

The question would also be, how will this behave if a user has less than 8 rooms in a given section?

Reply:
I originally wrote 8 but that looked weird so I removed that... Is this blocking the merge? Can we not just iterate on that depending on user feedback? 🙈 I could also write 8 in brackets?

When there are less rooms then nothing happens, I guess it would be better to remove the button in that case, is this also a blocker for the merge?

Reply:
Well I'm not entirely sure, I'd rather wait for a design review in here.
To me the wording brings an element of surprise, users do not entirely know what will happen until they click that button given that the default is not explicit

Also having the button do nothing when clicked seems that it would confuse user even more.

Not definitevely blocking, but let's wait for a design review and that should clear things up

Reply:
@gsouquet thanks for replying so fast.

What if I make it a label to start with? Does that make sense? 

Maybe we can have more text explaining in the label tab?

How about adding an explanation when hovering over the button?